### PR TITLE
feat(internet-header): add label to nav elements

### PR DESCRIPTION
**Pull request by alizedebray**

### CODE REVIEW

File: packages/internet-header/src/assets/i18n/translations.ts [hunk 1]
```diff
@@ -39,4 +39,9 @@ export const translations = {
     fr: 'Naviguer sur post.ch',
     it: 'Navigate su post.ch',
   },
+  'Intermediary links': {
+    de: 'Zwischenverbindungen',
+    fr: 'Liens intermédiaires',
+    it: 'Collegamenti intermedi',
```
Comment (oliverschuerch) [inline packages/internet-header/src/assets/i18n/translations.ts:45]:
```suggestion
  'Jump links': {
    de: 'Sprunglinks',
    fr: 'Liens de saut',
    it: 'Collegamenti di salto',
```

File: packages/internet-header/src/components/post-internet-breadcrumbs/components/middle-dropdown.component.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ export const MiddleDropdown = (props: {
       </button>
       <SvgIcon name="pi-arrow-down" classNames="rotate-270"></SvgIcon>
       {props.dropdownOpen ? (
-        <nav class="middle-dropdown">
+        <nav aria-label={translate('Intermediary links')} class="middle-dropdown">
```
Comment (oliverschuerch) [inline packages/internet-header/src/components/post-internet-breadcrumbs/components/middle-dropdown.component.tsx:26]:
```suggestion
        <nav aria-label={translate('Jump links')} class="middle-dropdown">
```

File: packages/internet-header/src/components/post-language-switch/components/post-language-switch-list.tsx [hunk 3]
```diff
@@ -7,8 +7,12 @@ export const PostLanguageSwitchList = (props: {
   switchLanguage: (lang: NavLangEntity) => void;
   dropdownRef: (element: HTMLElement | undefined) => void;
 }) => (
-  <nav class="language-switch-dropdown" ref={e => props.dropdownRef(e)}>
-    <h3 class="visually-hidden">{translate('Change language')}</h3>
+  <nav
+    class="language-switch-dropdown"
+    ref={e => props.dropdownRef(e)}
+    aria-labelledby="post-internet-header-language-switch-title"
```
Comment (oliverschuerch) [inline packages/internet-header/src/components/post-language-switch/components/post-language-switch-list.tsx:13]:
```suggestion
    aria-label={config.translations.navLangAriaLabel}
```

File: packages/internet-header/src/components/post-language-switch/components/post-language-switch-list.tsx [hunk 4]
```diff
@@ -7,8 +7,12 @@ export const PostLanguageSwitchList = (props: {
...
+  >
+    <h3 id="post-internet-header-language-switch-title" class="visually-hidden">{translate('Change language')}</h3>
     <ul>
```
Comment (oliverschuerch) [inline packages/internet-header/src/components/post-language-switch/components/post-language-switch-list.tsx:16]:
```suggestion
    <ul aria-label={translate('Change language')}>
```

File: packages/internet-header/src/components/post-internet-breadcrumbs/post-internet-breadcrumbs.tsx [hunk 5]
```diff
@@ -256,8 +256,9 @@ export class PostInternetBreadcrumbs {
               ></BreadcrumbList>
             </nav>
           </div>
-          <h2 class="visually-hidden">{breadcrumbConfig.a11yLabel}</h2>
+          <h2 id="post-internet-header-breadcrumb-title" class="visually-hidden">{breadcrumbConfig.a11yLabel}</h2>
           <nav
+            aria-labelledby="post-internet-header-breadcrumb-title"
```
Comment (oliverschuerch) [inline packages/internet-header/src/components/post-internet-breadcrumbs/post-internet-breadcrumbs.tsx:261]:
```suggestion
          <nav
            aria-label={breadcrumbConfig.a11yLabel}
```

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

File: packages/internet-header/src/assets/i18n/translations.ts (hunk 1 above)
Comment (oliverschuerch) [reply packages/internet-header/src/assets/i18n/translations.ts:45]: We could also go with "skip links" in english. It should be pretty well known by users.

Comment (oliverschuerch) [reply packages/internet-header/src/assets/i18n/translations.ts:45]: Sorry for all the changes,I didn't understand at the beginning what these translations were all about.

**Review by oliverschuerch — CHANGES_REQUESTED** [review]

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

**Pull request merged**